cvat/apps/quality_control/quality_reports.py

```diff
@@ -2303,21 +2303,22 @@ def _compute_reports(self, task_id: int) -> int:
             if validation_layout.mode == ValidationMode.GT_POOL:
                 task_frame_provider = TaskFrameProvider(task)
                 active_validation_frames = set(
-                    task_frame_provider.get_rel_frame_number(frame)
-                    for frame, real_frame in (
+                    task_frame_provider.get_rel_frame_number(abs_frame)
+                    for abs_frame, abs_real_frame in (
                         Image.objects.filter(data=task.data, is_placeholder=True)
                         .values_list("frame", "real_frame")
                         .iterator(chunk_size=10000)
                     )
-                    if real_frame in active_validation_frames
+                    if task_frame_provider.get_rel_frame_number(abs_real_frame)
+                    in active_validation_frames
                 )
 
             jobs: List[Job] = [j for j in job_queryset if j.type == JobType.ANNOTATION]
             job_data_providers = {
                 job.id: JobDataProvider(
                     job.id,
                     queryset=job_queryset,
-                    included_frames=set(job.segment.frame_set) & active_validation_frames,
+                    included_frames=active_validation_frames,
```

Comment:
I don't understand what you did here; why did you remove the intersection?

Reply:
Intersection is implemented inside JobDataProvider class
This intesection was incorrect because one set is with absolute frames, another one with relative frames

Reply:
https://github.com/cvat-ai/cvat/blob/63e43cad48a1d8da8e861a8555a924a173c0fbe0/cvat/apps/dataset_manager/bindings.py#L816